.travis.yml

```diff
@@ -11,10 +15,10 @@ env:
         - KAFKA_VERSION=0.8.2.0 SCALA_VERSION=2.11 PYTHONASYNCIODEBUG=0
 
 before_install:
-    - sudo apt-get install libsnappy-dev
-    - wget -N https://archive.apache.org/dist/kafka/${KAFKA_VERSION}/kafka_${SCALA_VERSION}-${KAFKA_VERSION}.tgz
-    - echo "Extracting kafka binaries for ${KAFKA_VERSION}"
-    - tar xzvf kafka_${SCALA_VERSION}-${KAFKA_VERSION}.t*
+    - sudo apt-get install -y libsnappy-dev
+    - sudo service docker stop
```

Comment:
We will get different bad behaviour if we use such hacks. The supported one is 1.7 - lets try to use that